config/application.yml.default

```diff
@@ -323,6 +323,7 @@ vendor_status_voice: 'operational'
 vendor_status_idv_scheduled_maintenance_start: ''
 vendor_status_idv_scheduled_maintenance_finish: ''
 verification_errors_report_configs: '[]'
+verify_by_mail_disabled_for_biometric_comparison: true
```

Comment:
I am getting a little tripped up by this being called `disabled`--could this be `verify_by_mail_enabled_for_biometric_comparison`?

Reply:
having a feature flag that defaults to false is very nice tho, makes rollouts easier?

Reply:
We recently expanded the documentation for feature flags to encourage `_enabled` as a suffix.

https://handbook.login.gov/articles/appdev-feature-flags.html#adding-a-new-feature-flag

Reply:
I just pushed https://github.com/18F/identity-idp/pull/10886/commits/1be59ee770664a0fc3c8ad4e22264a43ce62cf7e which renames this to `no_verify_by_mail_for_biometric_comparison_enabled`